### PR TITLE
Data Dict updates

### PR DESCRIPTION
**Pull request by jducnuigeen**
Fixes: #256 

### Change Description:

- Added image and description of building outline criteria
- Added a sentence to say effort has been made to adhere to outline criteria, but there may be updates later (as we find more issues not caught in QA).
- Removed wording in definition table to exclude mobile structures
- Added brief table descriptions
NOTE: published_data.rst will need to be updated with the correct final URL of the data, once we know it !

#### Testing Tasks:
- [ ] Added tests that fail without this change
- [ ] All tests are passing in development environment
- [x] Reviewers assigned
- [ ] Linked to main issue for ZenHub board

### CODE REVIEW

File: db/docs/source/introduction.rst [hunk 1]
```diff
@@ -72,7 +80,7 @@ The NZ Building Outlines dataset is being procured and released in stages. Image
 Accuracy Specification
 ---------------------------
 
-The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery to an accuracy of 1 metre. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees.
+The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery to an accuracy of 1 metre. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees. Effort has been made to adhere to the building outline criteria described above, and future updates may include adjustments of building outlines as imagery resolution allows.
 
 
 Valid Geometry
```
Comment (charlottedawson) [inline db/docs/source/introduction.rst]: the formatting for this looks strange - should it be bullet points?

File: db/docs/source/introduction.rst [hunk 2]
```diff
@@ -19,30 +19,38 @@ Over the next decade, the LINZ Topographic Office is working towards its vision
...
+The current Topo50 data for buildings is represented by either a `building polygon <https://data.linz.govt.nz/layer/50246-nz-building-polygons-topo-150k/>`_ or a `building point <https://data.linz.govt.nz/layer/50245-nz-building-points-topo-150k/>`_ . A building polygon is used to represent a structure that is large enough to be captured at 1:50,000 scale and has an area of 25m² or greater. The central business districts of large towns and cities is held in the LINZ building data as large building polygons. It is shown on the 1:50,000 scale printed maps as black fill, and is a generalized view, not showing individual buildings, or open spaces between buildings. A building point is used to represent a building with an area too small to be captured as a polygon feature at 1:50,000 scale.
```
[the remark this answers is not in the document]
Comment (jducnuigeen) [reply db/docs/source/introduction.rst]: Yep, agree, should be moved.

File: db/docs/source/introduction.rst [hunk 3]
```diff
@@ -19,30 +19,38 @@ Over the next decade, the LINZ Topographic Office is working towards its vision
...
+* Primary building structures are captured as separate building outline polygons from adjoining building structures, such as garages (see Image 2a)
+* Building extensions, sunrooms, balconies, patios and annexes are captured as part of the primary building outline structure if they exist as part of multi-level buildings and if sufficient imagery resolution allows this determination.
+* Permanent building structures such as sheds and greenhouses >10sq meters, not attached to a primary building structure, are captured as a separate building outline.
 * Adjoining townhouses are not captured as separate structures, but rather as joined primary structures.
-* Adjoining commercial buildings are captured as separate building outlines when rooflines allows delineation.
-* Building outline polygons captured will be greater than 10m².
-* Water tanks are captured as building outlines when their size is at least 16.5 square metres.
+* Adjoining commercial buildings are captured as separate building outlines when rooflines allows delineation (see Image 2c).
```
[the remark this answers is not in the document]
Comment (jducnuigeen) [reply db/docs/source/introduction.rst]: Yep, thanks.

File: db/docs/source/introduction.rst [hunk 4]
```diff
@@ -19,30 +19,38 @@ Over the next decade, the LINZ Topographic Office is working towards its vision
...
+
+The publicly available building outlines data consists of three tables. The `NZ_Building_Outlines <https://nz-buildings.readthedocs.io/en/latest/published_data.html#table-nz-building-outlines>`_ table contains all of the building outlines which match the most recent imagery available for each region of the country. The `NZ_building_outlines_full_history <https://nz-buildings.readthedocs.io/en/latest/published_data.html#table-nz-building-outlines-full-history>`_ table contains all building outlines that have been delineated from every imagery year/set that has been captured, whether this is historical imagery or the most recent imagery. The `NZ_building_outlines_lifecycle <https://nz-buildings.readthedocs.io/en/latest/published_data.html#table-nz-building-outlines-lifecycle>`_ table provides building relationship information as changes are detected between neighboring buildings.
```
[the remark this answers is not in the document]
Comment (jducnuigeen) [reply db/docs/source/introduction.rst]: Need to setup my sublime to correct for UK english. will investigate

File: db/docs/source/introduction.rst (hunk 1 above)
Comment (jducnuigeen) [reply db/docs/source/introduction.rst]: Changing this to mention accuracy between 90 and 110%

**Comment by Ben-Cheng** [comment]
Just to test something with Jan, ignore me

**Pull request merged**